addons/curator/XEH_postInit.sqf

```diff
@@ -8,6 +8,7 @@ if (isServer) then {
         INFO_1("Assigning Zeus to '%1'", _unit);
 
         private _curatorModule = [_unit] call FUNC(getFreeCuratorModule);
+        unassignCurator getAssignedCuratorLogic _unit;
```

Comment:
Is there any reason we first create new curator module (if free doesn't exist) and then unassign previous module?
```suggestion
        unassignCurator getAssignedCuratorLogic _unit;
        private _curatorModule = [_unit] call FUNC(getFreeCuratorModule);
```

Reply:
Yes, the idea is to use different module as I'm unsure if unassign and reassign of same module in single frame will fix the rare bug where you can't access the Zeus interface despite being assigned to module.

Reply:
I'll be merging this today if no more questions/suggestions.